internal/backends/linuxkms/display/egldisplay.rs

```diff
@@ -204,19 +204,33 @@ pub fn try_create_egl_display(
         })
         .ok_or_else(|| format!("No preferred or non-zero size display mode found"))?;
 
-    let encoder = connector
-        .encoders()
-        .iter()
-        .find_map(|handle| {
-            if connector.current_encoder() == Some(*handle) {
-                drm_device.get_encoder(*handle).ok()
-            } else {
-                None
-            }
-        })
-        .ok_or_else(|| format!("Not encoder found for connector"))?;
+    let encoder = connector.encoders().iter().find_map(|handle| {
+        if connector.current_encoder() == Some(*handle) {
+            drm_device.get_encoder(*handle).ok()
+        } else {
+            None
+        }
+    });
```

Comment:
I would have written something like:
```rust
let encoder = connector.current_encoder()
   .filter(|current| connector.encoders().iter().any(|h| current == h ))
   .and_then(|current| drm_device.get_encoder(*handle).ok());
```

But i say that without actually knowing the types of things and if that will event work. 
And this is also just a style issue, so not so important.

Reply:
Great, that works. Thanks!